Doc/library/dataclasses.rst

```diff
@@ -121,7 +121,7 @@ Module contents
      :meth:`__le__`, :meth:`__gt__`, or :meth:`__ge__`, then
      :exc:`TypeError` is raised.
 
-   - ``unsafe_hash``: If ``False`` (the default), a :meth:`__hash__` method
+   - ``unsafe_hash``: If false (the default), a :meth:`__hash__` method
```

Comment:
I think this would be "If false (the default is ``False``),", but I'm not sure that is an improvement. Somewhere it needs to say that "False" is the default. As it is, you've removed the actual default value.